tasks/queue-granules/index.js

```diff
@@ -62,6 +120,69 @@ function updateGranuleBatchCreatedAt(granuleBatch, createdAt) {
   }));
 }
 
+/**
+ * Constructs the partial params for enqueueGranuleIngestMessage. This is an optimization.
+ * This part of the message requires loading files out of S3. We don't want to do that everytime
+ * we create an SQS message considering that these parts at the same for every message, so this
```

Comment:
```suggestion
 * This part of the message requires loading files out of S3. We don't want to do that every time
 * we create an SQS message considering that these parts are the same for every message, so this
```
`everytime` should be `every time` and I think `at` is meant to be `are`?